grpc/sa-wrappers.go

```diff
@@ -490,12 +489,39 @@ func (sas StorageAuthorityClientWrapper) GetOrder(ctx context.Context, request *
 	return resp, nil
 }
 
+func (sas StorageAuthorityClientWrapper) GetAuthorizations(ctx context.Context, req *sapb.GetAuthorizationsRequest) (*sapb.Authorizations, error) {
+	resp, err := sas.inner.GetAuthorizations(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if resp == nil || resp.Authz == nil {
+		return nil, errIncompleteResponse
+	}
+	for _, element := range resp.Authz {
+		if element == nil || element.Domain == nil || !authorizationValid(element.Authz) {
+			return nil, errIncompleteResponse
+		}
+	}
+	return resp, nil
+}
+
+func (sas StorageAuthorityClientWrapper) AddPendingAuthorizations(ctx context.Context, req *sapb.AddPendingAuthorizationsRequest) (*sapb.AuthorizationIDs, error) {
+	resp, err := sas.inner.AddPendingAuthorizations(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if resp == nil || resp.Ids == nil {
+		return nil, errIncompleteResponse
+	}
+	return resp, nil
+}
+
 // StorageAuthorityServerWrapper is the gRPC version of a core.ServerAuthority server
 type StorageAuthorityServerWrapper struct {
-	inner *sa.SQLStorageAuthority
+	inner core.StorageAuthority
```

Comment:
Why did you change this from the concrete type to the interface? We are trying to go the opposite direction, to remove the hand-generated `core.*Authority` interfaces in favor of generated ones from protobufs.

Reply:
From the PR description:

> This also changes the inner type of grpc.StorageAuthorityServerWrapper to core.StorageAuthority so that we can avoid a circular import that is created by needing to import grpc.AuthzToPB and grpc.PBToAuthz (that latter of which needed to be exported) in `sa/sa.go`.

Reply:
Can we instead fix this by moving the FooToBar methods into a separate grpc/marshalling package?

Reply:
`grpc` would still need to import that package for the various wrappers files so we'd still get the `grpc <- grpc/marshalling <- sa <- grpc` loop.

Reply:
I don't want to hold up this change because of it, but this is a problem we're going to need to solve eventually. Can you file an issue describing the problem and we'll consider it ok for now?

Reply:
Reminder to file this issue.

Reply:
Ah yup, will do. Do you think we should include a TODO here referencing it?